### ISSUE TITLE
Wrong results on clicking button to display random chain

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#16]
##### This issue is about wrong result generated when `I'm feeling hype` button is clicked.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by Monal5031** [comment]
Prolly https://github.com/anshumanv/evolutions-webapp/blob/master/src/App.js#L29 is not working maybe we can do something of our own which can store the state?

**Comment by anshumanv** [comment]
Actually, it's not wrong, it's one step back, mean, it displays the result of the pokemon that was present in the text field right before the currently present.

**Comment by ong-yy** [comment]
Is anyone working on this? Can I?

**Comment by anshumanv** [comment]
@YernYee please work on it! :smile:

**Comment by ong-yy** [comment]
@anshumanv Alright, please give me some time. =)
